### ISSUE TITLE
support for transport.fly

### ISSUE THREAD

**Issue by basz** [pstadler/flightplan#66]
Is there a -native- way to run sub-tasks, something similar of 
```
transport.exec('fly build-phar:prod');
```
The above works, but ANSI colors are lost and we get nested output.
```
localhost ● ok
# transport.exec starts here
localhost $ fly build-phar:prod
localhost > ✈ Running build-phar:prod
localhost > ✈ Flight to localhost started
localhost > localhost $ echo Compiling Phar executable 
localhost > localhost > Compiling Phar executable
localhost > localhost ● ok
localhost > ✈ Flight to localhost finished after 18 ms
localhost > ✈ Flightplan finished after 19 ms
localhost ● ok
```

ps. Apologies if this is not the place for support. Where should I ask these questions?

**Comment by pstadler** [comment]
You could try to execute `plan.run(task, target, options)`, fly does the [same](https://github.com/pstadler/flightplan/blob/master/bin/fly.js#L95).

Please note that I never tried this, but I understand the use case, so please let me know if this works out for you.

**Comment by pstadler** [comment]
Another idea is to write your tasks like this:

```javascript
plan.local(['build-phar', 'deploy'], ...);
plan.local(['deploy'], ...);
```

When you now run `fly deploy`, the `build-phar` task is executed as well.

**Comment by ravi** [comment]
Had a long chat with @pstadler many centuries ago on a similar issue but now I cannot find it. I wanted to be able to do things: (1) compose tasks using other tasks, (2) be able to say: fly task1,task2,task3:destination .......

After various stops and starts, I have a framework built on top of Flightplan to obtain the above. Not entirely sure that this is the right way to go about it.

**Comment by basz** [comment]
`plan.run(task, target, options)` does work indeed. however the current task isn't halted. can't rely on its result therefore.

i'v opted for exec('fly ...'), for now. 
thx

**Comment by pstadler** [comment]
@basz thanks for reporting back.
@ravi see #45. I wouldn't call this a discussion, as you didn't explain me that the current implementation is not enough for your needs.

So there are two use cases:
1) Running multiple tasks from command line (e.g. `fly task1,task2:target`)
This could be done and makes sense somehow. Also the refactoring from 0.4.x to 0.5.x definitely makes this way easier to implement.

2) Running a task from a running task
Not sure about this one.

**Comment by ravi** [comment]
@pstadler fair enough, I didn't want to get too deep into the matter on #45 (thanks for finding that) because I figured the need may be specific to me.

**Comment by oliverzy** [comment]
Let me add case 3:
task1 has a dependency on task2, flightplan needs to provide a way to define this relationship.
